src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -477,8 +480,11 @@ describe('non-empty successful affected clusters table', () => {
             let sortedNames = _.map(filterData(filters), 'name');
             filterApply(filters);
             if (sortedNames.length === 0) {
-              // TODO check empty table view
-              // TODO headers are displayed
+              checkEmptyState(
```

Comment:
Given that we use the same title an body in multiple places, should we create a partial function for it?